src/views/ConsentContinuationOut.vue

```diff
@@ -0,0 +1,822 @@
+<template>
```

Comment:
This is an initial version only, copied from local corrections. There's lots that has to be updated in here, but it does save and load :)

There's another ticket to finish implementing this.